### PR TITLE
Whitelist known packages

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
Whitelist packages used by solutions accelerator

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known-readme.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Known libraries that are not in known.json
+
+1) libraries with Python syntax that astroid cannot parse
+tempo
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known-readme.txt:4]: has to be parseable. what are the errors?

File: src/databricks/labs/ucx/source_code/known-readme.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+mosaic
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known-readme.txt:5]: that's also our code

File: src/databricks/labs/ucx/source_code/known-readme.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+sparkdl
+chromadb
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known-readme.txt:7]: what's the problem with these two?

File: src/databricks/labs/ucx/source_code/known-readme.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+2) code that cannot be installed locally
+mediamix
+dbruntime
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known-readme.txt:11]: this is our internal package name

**Review by nfx — CHANGES_REQUESTED** [review]
> 29,895 additions, 1,247 deletions not shown because the diff is too large. Please use a local Git client to view these changes.

please split this PR into ~29 different ones - the way that we don't have more than 700-1000 lines in each? otherwise our release notes infra won't be able to pick it up. we want to have a proper changelog for libraries we've added here.

File: src/databricks/labs/ucx/source_code/known-readme.txt (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known-readme.txt:5]: I suspect an astroid bug when parsing comments with 'type' in the comment

**Comment by nfx** [comment]
closing this PR as it's impossible to review

**Pull request closed without merging**